File: conditionals/time.py
#sariah Kamore python

import time
first_time = time.gmtime()

current = time.time
#print(current)
now = time.ctime(current)
#print(now)
local_time = time.localtime(current)
day =  local_time.tm_wday
hour = local_time.tm_hour
print(day)
print(hour)

File: conditionals/time.c
//sariah kamore c
#include <stdio.h>
#include <time.h>

int main(void){
    //time since jan. 1st 1970
    time_t seconds;

    seconds = time(NULL);
    //printf("seconds since Jan. 1st 1970 = %d\n", seconds);
    time_t rawtime;
    struct tm *timeinfo;
    time(&rawtime);
    timeinfo = localtime(&rawtime);
   // printf("current time and date is %s\n", asctime(timeinfo));
   //current hour
   time_t now = time(NULL);
   struct tm *tm_struct = localtime(&now);
   int hour = tm_struct->tm_hour;
   printf("%d\n", hour);


    return 0;
}